README.md

```diff
@@ -0,0 +1,5 @@
+Some basic terms to avoid confusion between the different options/settings:
+
+- Setting: An about:config setting in firefox.
```

Comment:
s/Setting/Preference?

I think preference is less ambiguous.

Ok for everything else